app/styles/proposal.scss

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT license.
  */
 
-@import "bootstrap-sass-official/assets/stylesheets/_bootstrap.scss";
+@import "main.scss";
```

Comment:
dist css before:
186K main
21K vendor
after:
182K (!) main
21K vendor
Maybe the previous size was before some other changes...oh well :)